rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.62.0"
+components = [ "rustfmt", "clippy", "llvm-tools-preview" ]
```

Comment:
Is "llvm-tools-preview" necessary? I don't believe it is

Reply:
I took this from another project as an example, happy to slim this all down.